syntax = "proto3";
package datafilter;




import "resource_manage_sdk/model/resource_manage/filter_strategy_instance.proto";
import "resource_manage_sdk/model/resource_manage/filter_strategy_instance_data.proto";

/*
GetStrategyExecuteInstance请求
*/
message GetStrategyExecuteInstanceRequest {
    /*
    规则实例Id
    */
    string execId = 1 ;
}

/*
GetStrategyExecuteInstance返回
*/
message GetStrategyExecuteInstanceResponse {
    /*
    执行历史详情
    */
    resource_manage.FilterStrategyInstance instance = 1 ;
    /*
    不合规数据列表
    */
    repeated resource_manage.FilterStrategyInstanceData failDatas = 2 ;
}

/*
GetStrategyExecuteInstanceApi返回
*/
message GetStrategyExecuteInstanceResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    datafilter.GetStrategyExecuteInstanceResponse data = 4 ;
}

